frontend/components/Task.tsx

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+
+function Task() {
+	const [List, setList] = useState([])
+	const TaskList=async()=>{
```

Comment:
Please fix code formatting. Here's a quick read detailing [standard formatting rules](https://medium.com/nerd-for-tech/clean-code-formatting-source-code-structure-f3021575d79).